server/resourcemanager/resource_manager.go

```diff
@@ -340,7 +340,7 @@ func (m *manager[T]) list(w http.ResponseWriter, r *http.Request) {
 	}
 
 	listFn := m.rh.List
-	if isRequestForAugmented(r) {
+	if isRequestForAugmented(r) && m.rh.ListAugmented != nil {
```

Comment:
this fixes a panic on the server when getting an augmented list to a resource that have not registered a `ListAugmented` method